tokenomics/sagittarius-engine.md

```diff
@@ -0,0 +1,55 @@
+# Sagittarius Engine
+
+## Introduction
+The Sagittarius Engine is a module that incentivizes MKR holders to lock their MKR tokens up for long periods of time. The locked-up MKR is required to participate in Maker Governance via delegation. 
+
+The incentives for locking up MKR in the Sagittarius Engine are:
+- Token farming
+- Dai generation through vaults with MKR as collateral
+
+Withdrawing MKR from the Sagittarius Engine burns part of the deposited MKR.
+
+{% hint style="warning" %} This documentation describes planned functionality and processes that MakerDAO has not yet implemented. Be aware that parts may be inaccurate or out of date. {% endhint %}
+
+## Entry and Exit
+When users deposit MKR into the Sagittarius Engine, they must delegate the deposited MKR through the [Easy Governance Frontend (EGF)](../maker-core/easy-governance-frontend.md). Depositors must specify a delegate's strategy smart contract when making their deposit.
+
+When a user withdraws their MKR from the Sagittarius Engine, 15% of the deposited MKR is burned.
+
+## MKR Vaults
+The Sagittarius Engine allows users to use deposited MKR as collateral to generate Dai, similar to normal Maker Vaults. The parameters for these vaults are below:
+
+#### [Debt Ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) 
+There is no hard Debt Ceiling for these MKR vaults. However, a [Debt Ceiling Instant Access Module](https://manual.makerdao.com/module-index/module-dciam) will be configured to ensure that the Debt Ceiling does not increase too fast. A Soft Debt Ceiling parameter is set based on the value of all protocol-owned and SubDAO-owned assets.
+
+#### [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) 
+If the total Sagittarius Engine debt is less than the Soft Debt Ceiling, this is set to be equal to the [Dai Savings Rate](https://manual.makerdao.com/parameter-index/core/param-dai-savings-rate). It increases exponentially with the amount of Sagittarius Engine debt that exceeds the Soft Debt Ceiling.
```

Comment:
(The soft DC remains stationary despite applications of the DC-IAM an is changed through governance action, right?)

Reply:
Yes, correct